examples/demo-stf/src/tests/tx_revert_tests.rs

```diff
@@ -111,24 +118,28 @@ fn test_nonce_incremented_on_revert() {
     );
 
     {
+        // TODO: Maybe complete with actual block data
+        let _data = TestBlock::default();
         let mut demo = create_new_demo(path);
-        StateTransitionFunction::<MockZkvm, TestBlob>::init_chain(&mut demo, config);
+        StateTransitionFunction::<MockZkvm, TestBlob>::init_chain(&mut demo, config).unwrap();
 
-        let set_candidates_message = Runtime::<DefaultContext>::encode_election_call(
-            sov_election::CallMessage::SetCandidates {
-                names: vec!["candidate_1".to_owned(), "candidate_2".to_owned()],
-            },
-        );
+        let set_candidates_message =
+            <Runtime<DefaultContext> as EncodeCall<Election<DefaultContext>>>::encode_call(
```

Comment:
What is the advantage over the `encode_election_call`, the involved types are more complex now. Is it possible to to this change in another PR if it is not absolutely needed?